lib/web_ui/lib/src/engine/window.dart

```diff
@@ -573,8 +573,20 @@ final class EngineFlutterWindow extends EngineFlutterView implements ui.Singleto
 /// `dart:ui` window delegates to this value. However, this value has a wider
 /// API surface, providing Web-specific functionality that the standard
 /// `dart:ui` version does not.
-final EngineFlutterWindow window =
-    EngineFlutterWindow(kImplicitViewId, EnginePlatformDispatcher.instance);
+EngineFlutterWindow get window {
+  if (_window == null) {
+    throw StateError('Trying to access the implicit flutter view, but it has not been initialized.');
```

Comment:
Is there a scenario where in multi-view mode we _also_ have an implicit view? If not, maybe the error message should talk along the lines of "Implicit Flutter view is not available. Flutter is running in multi-view mode."

nit: s/flutter/Flutter/ (it's a proper name)

Reply:
> Is there a scenario where in multi-view mode we also have an implicit view?

I'm trying to avoid this, and only use the implicit view in the modes that exist right now (single view)

Reply:
Updated error message.